frontend/apps/reader/modules/readerfont.lua

```diff
@@ -71,24 +74,18 @@ end
 
 function ReaderFont:setupTouchZones()
     if Device:isTouchDevice() then
-        self.ui:registerTouchZones({
```

Comment:
That's old code. Actually this came up before so it would definitely be better to change it back, see https://github.com/koreader/koreader/pull/2768#discussion_r111660518